OpenIPSL/Electrical/Buses/Bus.mo

```diff
@@ -1,4 +1,4 @@
-﻿within OpenIPSL.Electrical.Buses;
+within OpenIPSL.Electrical.Buses;
```

Comment:
There is no modification here right?

Reply:
There was a UTF8 BOM that is deprecated and the CI is now checking for. So that was removed.